src/fidesops/service/connectors/saas_connector.py

```diff
@@ -120,13 +115,11 @@ def retrieve_data(
 
         rows: List[Row] = []
         for prepared_request in prepared_requests:
-            response = self.client().get(prepared_request)
+            response = self.client().send(prepared_request)
```

Comment:
What happens if you get a non-200 response in `retrieve_data`?

Reply:
Going to fix this in #182

Reply:
Fixed with changes merged in from main branch.